docs/rfcs/0002/readme.md

```diff
@@ -84,11 +120,14 @@ spec:
                 mountPath: /foo/persistent
               - name: nnf-config
                 mountPath: /nnf/config
+              ports:
```

Comment:
Instead of throwing everything into one example (which is overwhelming), I think every section should include just the details of that section.